modules/drivers/presto-jdbc/test/metabase/test/data/presto_jdbc.clj

```diff
@@ -24,6 +24,12 @@
 ;; during unit tests don't treat presto as having FK support
```

Comment:
I was getting `Catalog does not exist: database-name` when I tried to test presto-jdbc with my custom dataset, so I updated this to make all dataset for presto-jdbc to use the same catalog.

This is what we did for `presto` too.